travisfailed.py

```diff
@@ -232,16 +232,20 @@ def main(build_url, *, verbose=False, save_path='build_logs',
             print()
 
     if skipped and skipped_tests:
+        print()
         print('Skip Count / Tests')
         print('---------------------')
         for test, count in skipped_tests.items():
             print(f'{count} {test}')
         print()
 
     if count_failed:
+        print()
         print('Failure Count / Tests')
         print('---------------------')
-        for test, count in failed_tests.items():
+        for test, count in sorted(
```

Comment:
`sorted(sorted())`?

Reply:
The first one sorts by name, the second one (stably) sorts by fail count.  Do this is two steps because we want one of the sorts inverted.